grid.css

```diff
@@ -4,15 +4,15 @@ heading1, heading2{
 	width: 100%;
 	display: grid;
     grid-gap: 2%;	
-	grid-template-columns: 4% 26% 14% 22% 10% 7% 10%;
+	grid-template-columns: 4% 24% 14% 22% 11% 8% 10%;
 }
 
 /* style for the headings section */
 heading1{
 	padding: 0;
     border-bottom: 1px solid #aaa;
 	grid-template-rows: 50px;
-	background-color: white;
+	background-color: #527ECC;
```

Comment:
I changed the background colour to glaucous for the headings of the query playlists and the user list